WooCommerce/WooCommerceTests/POS/Presentation/Barcode Scanning/GameControllerBarcodeParserTests.swift

```diff
@@ -299,6 +299,150 @@ struct GameControllerBarcodeParserTests {
             }
         }
 
+        @Test("proactive timeout triggers error without next character")
+        func timeout_whenTimerFires_triggersTimeoutError() {
```

Comment:
super nit: the test function names feel like an interesting mix of snake & camel case 😆